src/components/core/DpEditor/DpEditor.stories.mdx

```diff
@@ -11,9 +11,42 @@ import DpEditor from './DpEditor'
 A rich text editor component based on tiptap.
 
 - defines a menubar with a number of buttons and an editor
-- use this component without the inline-editing-wrapper TiptapEditText.vue if you want to add a text editor to a form element (as in new statement view)
-- use this component with the inline-editing-wrapper TiptapEditText.vue if you want to save the text directly via inline-editing (as in assessment table)
-- the editor-content component needs a prop with editor instance to work correctly
 
 To properly set the content we have to update this.currentValue and use editor.setContent() - both actions are needed!
-To use boilerplates we need to mount tiptap editor, where boilerplate modal is imported dynamically. The boilerplates are stored in boilerplates vuex store, which is also registered dynamically if boilerplate prop is specified.
+
+To use image upload we need to add imageButton optional Prop.
+
+To use another component within DpEditor, we can add Slot named modal.
```

Comment:
This implies that DpEditor can be easily extended using slots. This is not the case - `v-slot:modal` is still tightly coupled to the inner workings of the boilerplate feature. I'd suggest no not mention the slot functionality at all, at least before we have a fully working way of extending DpEditor.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/567/commits/e6899199500bcb6290964231adf96b76dfe2a5b4
Sorry, wrong commit naming

Reply:
i meant to not mention it at all.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/567/commits/2fc10b706e8b427d0a2e8ff420e209799380f9b7
see hier, I added also missing props!